### PR TITLE
Add 2020-02-02-preview related to new ImportExport API

### PR DESCRIPTION
**Pull request by brandong-ms**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by jianyexi** [comment]
@brandong-ms 
Since the `~[Staging] Swagger LintDiff` has passed , the other lintdiff can be ignored . Because the `~[Staging] Swagger LintDiff` the latest version of LintDiff. If the PR merged is blocked by the the LintDiff at last,  you can ask @akning-ms  to force merge.

**Review by jamestao — APPROVED** [review]

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json
Comment (pilor) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json:351]: why not the full privateEndpointConnection resource ID?  Names of resources are typically not good ways to reference something as it requires the consumer to reconstruct the resource ID themselves

**Review by pilor — CHANGES_REQUESTED** [review]

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json
Comment (brandong-ms) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json:19]: This change is to make it more consistent since servers import and export are actions directly on the object. I am not sure about the history of this API and why we used the extensions path. 

Also, for the customers who are still using the old format of API, it will still work if they use the 2014 version, just not the 2020 version. So it should not be a problem AFAIK, because anyone using the new version will be writing new code. Let me know if that's not acceptable, and what is the usual way to provide the compatibility for these situations.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json
Comment (brandong-ms) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json:273]: Setting this value indicates that a private link should be created for the resource ID. The customer may leave this value empty, but still want to create a private link for the storage account. The API supports creating private link for either SQL server, or storage account, or both.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json
Comment (brandong-ms) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json:351]: That's a good point, I think it would be worthwhile to add the full resource name. However, given this feature is urgent, would it be possible for me to go back and add this later? I could add it as a new property so it would not be breaking. If that works for you I'll open a work item to add the privateEndpointConnectionResourceId property.

Comment (brandong-ms) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json:351]: Opened a task for this:

Defect 13694433:Add PrivateEndpointConnectionResourceId property to PrivateEndpointConnectionRequestStatus
http://sqlbuvsts01:8080/Main/SQL%20Server/_workitems#id=13694433&_a=edit

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json
Comment (pilor) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json:19]: Changing the URI like this breaks RBAC users have set up to give people access to import operations since it is now a new action that requires a different privilege.  Its up to your service and customers whether thats appropriate given the -preview nature of these APIs (although the 2014 API has been around for years)

File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json
Comment (pilor) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importexport.json:351]: Its not ideal but since this is a -preview API and its a status response its ok for now

**Review by pilor — APPROVED** [review]
Approving with a cautionary note about the change in /import URI from the old api-version

**Comment by brandong-ms** [comment]
Hey Chris,

I have discussed with my team, and we are ok if we have one-time cost to change permissions for users upgrading to new API. The old API version should still be available for a while for users who don’t want to do this right away.

Can you please merge the PR?

Thanks,
Brandon

**Comment by pilor** [comment]
The SDK reviewer (@erich-wang) needs to review and merge.  API reviewers don't merge

**Review by erich-wang — APPROVED** [review]

**Review by erich-wang — APPROVED** [review]

**Comment by erich-wang** [comment]
I was told that we could ignore `public.rest-api-specs (LintDiff)` failure if `~[Staging] Swagger LintDiff ` passes, @akning-ms could please force merge this PR?

**Pull request merged**
